client/a8c-for-agencies/sections/team/primary/team-invite/index.tsx

```diff
@@ -48,20 +60,38 @@ export default function TeamInvite() {
 							duration: 5000,
 						} )
 					);
+					dispatch( recordTracksEvent( 'calypso_a4a_team_invite_success' ) );
 					page( `${ A4A_TEAM_LINK }/${ TAB_INVITED_MEMBERS }` );
 				},
 
 				onError: ( error ) => {
 					dispatch( errorNotice( error.message ) );
+					dispatch(
+						recordTracksEvent( 'calypso_a4a_team_invite_error', {
+							error: 'api_error',
+						} )
+					);
```

Comment:
Same as above. Not sure if this is needed. What do you think?

Reply:
In this case, I could see a possible utility, like detecting a peak of API errors.